test/integration/connectors/test_milvus.py

```diff
@@ -139,3 +143,26 @@ async def test_milvus_destination(
     uploader.run(path=staged_filepath, file_data=file_data)
     with uploader.get_client() as client:
         validate_count(client=client, expected_count=expected_count)
+
+
+@pytest.mark.tags(CONNECTOR_TYPE, DESTINATION_TAG)
+class TestPrecheck:
```

Comment:
Why are we introducing a class based approach here?

Reply:
Can we stick to single test methods to keep things consistent?

Reply:
I've refactored this into two separate functions, good point!